### PR TITLE
Enhance monitor app

### PR DESCRIPTION
**Pull request by kongzii**
Screenshot by Dropbox Capture

### CODE REVIEW

File: prediction_market_agent_tooling/markets/manifold/data_models.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def clip_timestamp(cls, value: int) -> datetime:
         """
         Clip the timestamp to the maximum valid timestamp.
         """
-        max_timestamp = (datetime.max - timedelta(hours=1)).timestamp()
+        max_timestamp = (datetime.max - timedelta(days=1)).timestamp()
```
Comment (kongzii) [inline prediction_market_agent_tooling/markets/manifold/data_models.py:82]: bug fix, for some reason I started to have `year 10000 is not valid` error

File: prediction_market_agent_tooling/monitor/monitor.py [hunk 2]
```diff
@@ -203,16 +203,43 @@ def monitor_agent(agent: DeployedAgent) -> None:
...
+        .interactive()
+    )
+    st.altair_chart(
+        per_day_accuracy_chart.mark_line()
+        + per_day_accuracy_chart.transform_loess("x-axis-day", "Is Correct").mark_line(
+            color="red", strokeDash=[5, 5]
+        ),
+        use_container_width=True,
+    )
+
```
[the remark this answers is not in the document]
Comment (kongzii) [reply prediction_market_agent_tooling/monitor/monitor.py:239]: Why? Altair was used here before as well, and it has a nice integration with streamlit. And it's interactive (you can zoom in, out, etc. on the plot).

[the remark this answers is not in the document]
Comment (gabrielfior) [reply prediction_market_agent_tooling/monitor/monitor.py:239]: I just think we should default to one library, and when I run
```
df.plot(x,y)...
```
this uses Matplotlib by default. 
So I don't have a strong opinion on which plotting library we use, but I would prefer to have only one across all plots.

[the remark this answers is not in the document]
Comment (kongzii) [reply prediction_market_agent_tooling/monitor/monitor.py:239]: With that logic, we should use Altair, shouldn't we? Evan already used it in this monitoring app, and I can't find any usage of matplotlib anywhere in PMAT:

Screenshot by Dropbox Capture

Screenshot by Dropbox Capture

Screenshot by Dropbox Capture

[the remark this answers is not in the document]
Comment (gabrielfior) [reply prediction_market_agent_tooling/monitor/monitor.py:239]: Sure! Altair as standard works for me.

**Review by gabrielfior — APPROVED** [review]

**Pull request merged**